spec/features/stripe_customer_spec.rb

```diff
@@ -0,0 +1,176 @@
+# frozen_string_literal: true
+
```

Comment:
The first 100 lines here are still almost identical to the ones in stripe_checkout.spec. I think that all this duplication may become an issue in the future, as changes in one file will need to be ported to the other, for example if something changes on Solidus FE flow.

Reply:
Good point. I’ve moved this to a `checkout_helper`, into a method called `initialize_checkout`. This deduplicates almost all of the code that was shared between `stripe_checkout_spec` and  `stripe_customer_spec`.